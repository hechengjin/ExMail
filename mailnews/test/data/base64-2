Content-Type: text/html; encoding=iso-8859-1
Content-Transfer-Encoding: base64

PGh0bWw+PGJvZHk+VGhpcyBpcyBiYXNlNjQgZW5jb2RlZCBIVE1MIHRleHQsIGFuZCB0aGUgdGFncyB
zaG91bGRuJ3QgYmUgc3RyaXBwZWQuDQo8Yj5Cb2xkIHRleHQgaXMgYm9sZCE8L2I+PC9ib2R5PjwvaH
RtbD4NCg==
