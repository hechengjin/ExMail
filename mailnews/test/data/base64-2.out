<html><body>This is base64 encoded HTML text, and the tags shouldn't be stripped.
<b>Bold text is bold!</b></body></html>